server/game/cards/06-WoE/Conscription.js

```diff
@@ -0,0 +1,38 @@
+const Card = require('../../Card.js');
+
+class Conscription extends Card {
+    // Play: If there are more enemy creatures than friendly
+    // creatures, make a number of token creatures equal to the
+    // difference.
+    setupCardAbilities(ability) {
+        this.play({
+            gameAction: ability.actions.conditional({
+                condition: (context) =>
+                    !!context.player.opponent &&
+                    context.player.opponent.creaturesInPlay.length >
+                        context.player.creaturesInPlay.length,
+                trueGameAction: ability.actions.makeTokenCreature((context) => ({
+                    amount:
+                        context.player.opponent.creaturesInPlay.length -
+                        context.player.creaturesInPlay.length
+                }))
+            }),
+            effect: 'make {1} token creature{2}',
```

Comment:
I believe this is the default effect msg

Reply:
Unfortunately the `conditional` action replaces it with `do something conditionally` with no other info.  I didn't see a way to turn that off but let me know if there is.

Reply:
You can remove the condition and make `amount = 0` when the condition is not applicable
```
this.play({
                gameAction: ability.actions.makeTokenCreature((context) => ({
                    amount: !!context.player.opponent &&
                    context.player.opponent.creaturesInPlay.length >
                        context.player.creaturesInPlay.length ? 
                        context.player.opponent.creaturesInPlay.length -
                        context.player.creaturesInPlay.length : 0
                }))
            }),
```

Reply:
Fixed along with Election, thanks!